### PR TITLE
Code for getting archiveNumbers using headers

### PR DESCRIPTION
**Pull request by aishwaryakapse**
Got Archive numbers in a set for page 1. Need to get number of pages to write similar queries for other pages. 

Need to incorporate this code with scrapy code to start crawling all the pages with the archive numbers in the set.

#66 

@JavierJia @chenlica @MoniMoledo @kaiyim

### CODE REVIEW

File: noah/src/main/resources/crawler/promed_mail.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	response = urllib2.urlopen(request)
+	return response
+
+def getQuery1Page1(start_date, end_date):
+	query1 = {"kwby1":'summary', 'search':'zika', 'show_us':'1', 'feed_id':'1'}
+	query1['date1'] = start_date
+	query1['date2'] = end_date
+	return query1
+	
+def getQuery2Page2(start_date, end_date):
```
Comment (JavierJia) [inline noah/src/main/resources/crawler/promed_mail.py]: all those `getQuery*page*` is exactly the same. should it be one function just called `getNext`?

File: noah/src/main/resources/crawler/promed_mail.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	return listId
+
+if __name__ == '__main__':
+	
+	archNo = set([])
+	# get today's date
+	end_date = datetime.date.today()
+	start_date = datetime.datetime(2007, 01, 01)
+	end_date = end_date.strftime("%m/%d/%Y")
+	start_date = start_date.strftime("%m/%d/%Y")
```
Comment (JavierJia) [inline noah/src/main/resources/crawler/promed_mail.py]: too verbose. it's better to merge two assignment.
`start_date =datetime.datetime(2007, 01, 01).strftime("%m/%d/%Y")`

File: noah/src/main/resources/crawler/promed_mail.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+	filename = "./promedsearchresult.html.gzip"
```
Comment (JavierJia) [inline noah/src/main/resources/crawler/promed_mail.py]: it should be a parameter.

Comment (JavierJia) [inline noah/src/main/resources/crawler/promed_mail.py]: could you use [some lib](http://stackoverflow.com/questions/1543652/python-gzip-is-there-a-way-to-decompress-from-a-string) to unzip it from string other than the gzip file? it is too expensive.

File: noah/src/main/scala/edu/uci/ics/cloudberry/noah/crawler/promed_mail.scala [hunk 4]
```diff
@@ -0,0 +1,8 @@
+package edu.uci.ics.cloudberry.noah.crawler
```
Comment (JavierJia) [inline noah/src/main/scala/edu/uci/ics/cloudberry/noah/crawler/promed_mail.scala]: remove this file.

**Comment by JavierJia** [comment]
what should we do with this PR???

**Comment by aishwaryakapse** [comment]
I am extremely sorry. This code was wrong. I by-mistake deleted this branch. could you please delete this pull request?

**Comment by JavierJia** [comment]
no worries. you can close this PR.

**Pull request closed without merging**